src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/CaseloadService.kt

```diff
@@ -19,24 +20,32 @@ class CaseloadService(
   fun getPrisonersByNumber(nomisIds: List<String>) =
     prisonerSearchApiClient.searchPrisonersByNomisIds(nomisIds).map { it.toCaseloadItem() }
 
-  fun getPrisonersByReleaseDate(earliestReleaseDate: LocalDate, latestReleaseDate: LocalDate, prisonIds: Set<String>, page: Int = 0) =
-    prisonerSearchApiClient.searchPrisonersByReleaseDate(earliestReleaseDate, latestReleaseDate, prisonIds, page).map { it.toCaseloadItem() }
+  fun getPrisonersByReleaseDate(
+    earliestReleaseDate: LocalDate,
+    latestReleaseDate: LocalDate,
+    prisonIds: Set<String>,
+    page: Int = 0,
+  ): Page<CaseloadItem> =
+    prisonerSearchApiClient.searchPrisonersByReleaseDate(earliestReleaseDate, latestReleaseDate, prisonIds, page)
+      .map { it.toCaseloadItem() }
 
   fun getPrisoner(nomisId: String) =
     getPrisonersByNumber(listOf(nomisId)).firstOrNull() ?: throw EntityNotFoundException(nomisId)
 
   private fun PrisonerSearchPrisoner.toCaseloadItem(): CaseloadItem {
     val sentenceDateHolder = this.toSentenceDateHolder()
     return CaseloadItem(
-      prisoner = this.toPrisoner(),
+      prisoner = this,
```

Comment:
**question**: why did these files change?

Reply:
oh.. you're trying to use it in the CaCaseloadService.. I would suggest just calling the client directly in your caseload service rather than trying to reuse this

Reply:
fixed it.